### PR TITLE
[Chore] Reuse the reusable code parts

### PR DESCRIPTION
**Pull request by FrankEssenberger**
## Context

We had already example and documentation for the OData v2. Try to reuse this also for v4 where possible. 
I would like to have better blocks of documentation but the problem with the TOC and the headers blocks this.

So this is the best I could do.

### CODE REVIEW

File: docs/js/features/odata/common/query-parameter/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export function Top(){
+  return <TopContent/>
+}
+
+export function Skip(){
+  return <SkipContent/>
+}
+
+export function Select(){
```
Comment (jjtang1985) [inline docs/js/features/odata/common/query-parameter/index.jsx:16]: Is this a v2 item? the import is missing

Comment (FrankEssenberger) [reply docs/js/features/odata/common/query-parameter/index.jsx:16]: oh yes. v2 has a select item and v4 a expand-select item which are rather different.

File: docs/js/features/odata/odata-v4-client.mdx [hunk 2]
```diff
@@ -13,112 +13,92 @@ keywords:
...
+
+  #### Filter on One-To-Many Navigation Properties
+  <FilterOneToMany/>
+
+  #### More Filter Expressions
+  More advanced filter expressions can be found [here](#available-filter-expressions).
+
+  ### Expand
+
+  #### Expand and Select
```
Comment (jjtang1985) [inline docs/js/features/odata/odata-v4-client.mdx:50]: I think `$select` should have the same level as `$expand`/`$filter`

File: docs/js/features/odata/odata-v4-client.mdx [hunk 3]
```diff
@@ -13,112 +13,92 @@ keywords:
...
+  ### Deep Create
+  <DeepCreate/>
+
+  ### Create as Child of
+  <CreateAsChildOf/>
+
+## Update Request Builder
+<Update/>
+
+## Delete Request Builder
```
Comment (jjtang1985) [inline docs/js/features/odata/odata-v4-client.mdx:80]: Etag and csrf token are missing.

**Review by jjtang1985 — APPROVED** [review]
I like the way how we "reuse" the doc with the `jsx` files.
Please consider some minor suggestions.

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**